README.md

```diff
@@ -137,6 +137,22 @@ const { total } = await myDocumentDao.decr(
 );
 ```
 
+When multiple values must be incremented and/or decremented in the same call:
```

Comment:
I don't think the function should have been named `multiIncr` if it is also capable of decrementing, since the `incr` and `decr` functions are separate. Perhaps something like `updateCounters` would have been better.